modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OLSLoad_conf.pm

```diff
@@ -222,8 +221,8 @@ sub pipeline_analyses {
             -max_retry_count => 1,
             -flow_into       => {
                 1 => WHEN(
-                    '#skip_phi#' => [ 'compute_closure' ],
-                    ELSE [ 'phibase_load_factory' ]
+                    '("PHI" ~~ #ontologies# and (#wipe_one# == 1 or #wipe_all# == 1))' => [ 'phibase_load_factory' ],
```

Comment:
I've steered clear of smartmatching in Perl - too smart for my liking, and since it's still an experimental feature, the warnings are annoying, if nothing else...
An alternative is to use Perl's `grep` method, which I've always thought was a misleading use of nomenclature... haven't tested it in the context of this pipeline, but the following should do the same thing, I think:
 ```suggestion
                    '( #expr(grep {/PHI/} @{#ontologies#})expr# and (#wipe_one# == 1 or #wipe_all# == 1) )' => [ 'phibase_load_factory' ],
```

Reply:
I guessed this was the reason behind the inactivation of fatal?

Reply:
I've tried with that syntax but it doesn't seems to be working, as I get the following error:
```
ParamError: Cannot evaluate the expression: 'expr(( #expr(grep {/PHI/} @{#ontologies#})expr# and (#wipe_one# == 1 or #wipe_all# == 1) ))expr' ==> '( #expr(grep {/PHI/} @{$self->_param_possibly_overridden('ontologies', $overriding_hash)})expr# and ($self->_param_possibly_overridden('wipe_one', $overriding_hash) == 1 or $self->_param_possibly_overridden('wipe_all', $overriding_hash) == 1) )' syntax error at (eval 426) line 2, at EOF
```

Reply:
```suggestion
                    '(grep {/PHI/} @{#ontologies#} and (#wipe_one# == 1 or #wipe_all# == 1))' => [ 'phibase_load_factory' ],
```
Sorry, got the syntax wrong, didn't need the #expr because that's already implied by the enclosing parentheses.